src/main/webapp/dataset.xhtml

```diff
@@ -1810,27 +1810,30 @@
                         <p class="text-warning">
                             <span class="glyphicon glyphicon-warning-sign"/> #{bundle['dataset.rejectMessage']}
                         </p>
-                        <ui:remove>
-                            <!--FIXME update for new comments table -->
-                            <p:inputTextarea id="returnReason" rows="3" value="#{DatasetPage.workingVersion.returnReason}" title="#{bundle['dataset.rejectMessage.label']}" maxlength="200" widgetVar="returnReason"
-                                             cols="70" counter="display" counterTemplate="{0} characters remaining." autoResize="false"
-                                             requiredMessage="#{bundle['dataset.reject.enterReason.error']}"/>
-                            <!--FIXME validation error msg for returnReason was in confirmDialog using testReturnReason() in commandButton below but all that was removed
-                                      as it did not look like the usual validation method, added preferred requiredMessage attribute to inputTextarea above, need wiring #5717-->
-                            <p>
-                                <h:outputText id="display"/>
-                            </p>
-                            <p:watermark for="returnReason" value="#{bundle['dataset.rejectWatermark']}" id="returnReasonwatermark"/>
-                        </ui:remove>
+
+                        <p:inputTextarea id="returnReason" rows="3" value="#{DatasetPage.returnReason}" title="#{bundle['dataset.rejectMessage.label']}" maxlength="200" widgetVar="returnReason"
+                                         cols="70" counter="display" counterTemplate="{0} characters remaining." autoResize="false"
```

Comment:
From @sbarbosadataverse via [Slack](https://iqss.slack.com/archives/CVB2SMDFX/p1701199338230539):

"can up we that character limit on the reason for return?"

Reply:
`maxlength="200"` can be increased, how many do you feel good enough @sbarbosadataverse ?

Reply:
To give an example, at INRAE, our current emails of feedback when curating mimicking a return to author are currently around 1200 characters.

Reply:
Here's an example for context, in french to keep the original length :
```
Bonjour,

Afin d’assurer un certain niveau de qualité de l’espace institutionnel Data INRAE, l’équipe support réalise régulièrement un contrôle des jeux de données qui y sont publiés, portant essentiellement sur les métadonnées et la documentation associée. A ce titre, nous vous suggérons quelques améliorations du jeu de données «The Enigmatic Chronicles of the Celestial Realm» https://doi.org/10.12345/ABCDEF, dont vous êtes la personne-contact.

Vous trouverez ces recommandations dans le rapport de curation disponible via ce lien. Nous vous encourageons à les suivre afin d'améliorer la visibilité et la réutilisation de votre jeu de données. Nous avons effectué nous-mêmes un certain nombre de modifications simples, indiquées dans le rapport.

Nous vous rappelons que toute modification entraine la création d’un nouveau brouillon du jeu de données qui doit être re-publié une fois l’ensemble des corrections apportées.

N'hésitez pas à revenir vers nous pour toute question en nous écrivant à l'adresse suivante someemail@email.fr  

Bien cordialement,

Someone Someone
Pour Data INRAE, l’espace INRAE de l’entrepôt Recherche Data Gouv
```

Reply:
maxlength increasedto 2000.